websites/O/OK/presence.ts

```diff
@@ -0,0 +1,203 @@
+const presence = new Presence({
+	clientId: "1036732879725658213",
+});
+
+enum Assets {
+	friends = "Друзья",
+	friendRequests = "Друзья",
+	pymk = "Друзья",
+	outgoingFriendRequests = "Друзья",
+	photos = "Фото",
+	statuses = "Заметки",
+	groups = "Группы",
+	market = "Товары",
+	music = "Музыка",
+	games = "Игры",
+	video = "Видео",
+	giftsFriend = "Подарки",
+	about = "О себе",
+	feed = "новости",
+	guests = "гостей",
+	marks = "события",
+	messages = "сообщения",
+	vitrine = "игры",
+	gifts = "подарки",
+	discovery = "рекомендации",
+	topphoto = "фото-конкурс",
+	marathons = "марафоны",
+	services = "приложения",
+	mall = "товары",
+	payments = "платежи и подписки",
+	online = "людей на сайте",
+}
+
+async function getStrings() {
+	return presence.getStrings({
+		playVideo: "general.watchingVid",
+		playMusic: "general.playing",
+		pause: "general.paused",
+	});
+}
+
+let video = {
+		duration: 0,
+		currentTime: 0,
+		paused: true,
+	},
+	strings: Awaited<ReturnType<typeof getStrings>>;
+
+function textContent(tags: string) {
+	return document.querySelector(tags)?.textContent;
+}
+
+function typeContent(string: string) {
+	return Assets[string as keyof typeof Assets];
+}
+
+function getMillisecondsFromString(timeString: string): number {
+	const parsedText = timeString?.split(":");
+	return (Number(parsedText[0]) * 60 + Number(parsedText[1])) * 1000;
```

Comment:
Resolve CI.
Possible solutions:
* Remove optional chaining on `const parsedText = ...`
* Add optional chaining to `return (Number(....`

Reply:
> Resolve CI.

Now it is better?

Reply:
Looks better I think.